File: DDS_Restaurant_Solution/DDS_Restaurant_Solution/Models/Sucursal.cs
    using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace DDS_Restaurant_Solution.Models
{
    public class Sucursal
    {
        [Key]
        public int idSucursal { get; set; }

        [MaxLength(15)]
        public string establecimiento { get; set; }

        [MaxLength(15)]
        public string nombreSucursal { get; set; }

        public string plantas { get; set; }

        public string direccion { get; set; }

        [MaxLength(8)]
        public string telefono { get; set; }

        [MaxLength(50)]
        public string correo { get; set; }

        [MaxLength(15)]
        public string RTN { get; set; }
    }
}

File: DDS_Restaurant_Solution/DDS_Restaurant_VB/Models/Sucursal.vb
Imports System.ComponentModel.DataAnnotations.Schema
Imports System.ComponentModel.DataAnnotations
Public Class Sucursal
    <Key>
    Public Property idSucursal As Integer
    <MaxLength(15)>
    Public Property establecimiento As String
    <MaxLength(15)>
    Public Property nombreSucursal As String
    Public Property plantas As String
    Public Property direccion As String
    <MaxLength(8)>
    Public Property telefono As String
    <MaxLength(50)>
    Public Property correo As String
    <MaxLength(15)>
    Public Property RTN As String
End Class